src/test/java/uk/gov/pay/webhooks/webhook/WebhookServiceTest.java

```diff
@@ -110,5 +111,27 @@ public void shouldReturnSubscribedWebhooksForGivenEventTypeServiceIdLivenessTupl
         var subscribedWebhook = subscribedWebhooks.get(0);
         assertThat(subscribedWebhook.getSubscriptions(), hasItem(capturedEventType));
     }
+
+    @Test
+    public void shouldReturnSigningKey() {
+        when(idGenerator.newWebhookSigningKey(live))
+                .thenReturn("some-signing-key");
+        
+        var createWebhookRequest = new CreateWebhookRequest(
+                serviceId,
+                live,
+                callbackUrl,
+                null,
+                null
+        );
+
+        webhookService.createWebhook(createWebhookRequest);
+
+        ArgumentCaptor<WebhookEntity> argumentCaptor = ArgumentCaptor.forClass(WebhookEntity.class);
+        verify(webhookDao).create(argumentCaptor.capture());
+        WebhookEntity captured = argumentCaptor.getAllValues().get(0);
+        assertThat(captured.getSigningKey(), equalTo("some-signing-key"));
```

Comment:
This should work now:

```java
verify(webhookDao).create("some-signing-key");
```

Reply:
Doesn't look like that syntax works as `create` takes a `WebhookEntity`

Reply:
Ah, yes, you’re right.